packages/strapi-plugin-content-manager/config/policies/routing.js

```diff
@@ -8,7 +8,40 @@ module.exports = async (ctx, next) => {
 
     if (controller && action) {
       // Redirect to specific controller.
-      return await strapi.plugins[source].controllers[controller.toLowerCase()][action](ctx);
+      if (ctx.request.body.hasOwnProperty('fields') && ctx.request.body.hasOwnProperty('files')) {
+        const {files, fields} = _.cloneDeep(ctx.request.body);
+
+        const fileAttributes = _.get(strapi.plugins, [source, 'models', ctx.params.model, 'associations'], []).filter(association => {
+          return association[association.type] === 'file' && association.via === 'related';
+        }).reduce((acc, association) => {
+          acc.push(association.alias);
+          return acc;
+        }, []);
+
+        ctx.request.body = _.omit(fields, fileAttributes);
```

Comment:
Not sure to understand why you're omitting file relationships... What's happening if I'm removing a file from a field which can take multiple files. I think we need these fields to be able to remove the relationships.

Reply:
Exact we can't delete files when it's multiple file ! But actually if I send relationships, it's break the relation between files. I will investigate.